docs/cold_storage_test.py

```diff
@@ -0,0 +1,312 @@
+# jupyter:
+#   kernelspec:
+#     name: my-poetry-env
+#     display_name: Python 3
+#   language_info:
+#     name: python
+# ---
+
+# # Cold Storage
+#
+# Por padrão, novos objetos utilizam a classe de armazenamento "standard", que é adequada para acessos frequentes.
+# No entanto, para muitos casos, o acesso aos objetos armazenados pode ser raro e o principal objetivo é garantir que eles
+# sejam mantidos por longos períodos, ainda que disponíveis para acesso rápido quando necessário.
+#
+#  Exemplos: backups, logs, registros arquivados para cumprimento de legislações.
+
+# Para esses casos, nas regiões suportadas, está disponível a classe de armazenamento fria "cold_instant",
+# que oferece um custo de armazenamento reduzido e um custo de acesso mais elevado. Consulte a página de Preços
+# para comparar as diferentes classes de armazenamento.
+
+# + tags=["parameters"]
+config = "../params/br-ne1.yaml"
```

Comment:
como só suportamos no Sudeste 1, seria legal este exemplo vir com uma config que funcione

```suggestion
config = "../params/br-se1.yaml"
```